lib/npg_warehouse/loader/pacbio/run.pm

```diff
@@ -174,6 +180,15 @@ sub _build_run_wells {
       my $tki = $self->_well_tracking_info($well->{'uniqueId'});
       my $run = $self->_run;
 
+      open my $id_product_script, q[-|],
+        join q[ ], 'python', $self->id_script,
```

Comment:
Reading from the output of the process is slightly different from reading from a file handle. We have to deal with a case when the process errors.